doc/dynamic-environments/configuration.mkd

```diff
@@ -121,6 +121,10 @@ sources:
     basedir: '/etc/puppet/environments'
     prefix: true # All environments will be prefixed with "mysource_"
 ```
+#### prefix behaviour
+* if `true` environment folder will be prefixed with the `source` name.
```

Comment:
This reads as if it's literally "source" - might be good to rephrase to something like "the name of the source".